src/apps/settings/src/app/views/Robots/Robots.less

```diff
@@ -33,9 +33,9 @@
     margin-bottom: 8px;
   }
 
-  .SaveBtn {
```

Comment:
Don't see these styles ported over. Are they no longer necessary on the button?

Reply:
Not needed.  This was fixed by adding `sx={{ alignSelf: "flex-start" }}` to the button itself.